cookbook/deployment/platformsh.rst

```diff
@@ -37,9 +37,11 @@ Platform.sh how to deploy your application (read more about
 
     # The name of this app. Must be unique within a project.
     name: myphpproject
-
-    # The toolstack used to build the application.
-    toolstack: "php:symfony"
+    
+    # The type of the application to build.
+    type: php:5.4
```

Comment:
Giving a version number here does work? Just asking as it is not explained in the docs.